src/components/PlaceholderContainer/README.md

```diff
@@ -162,7 +162,7 @@ with src and alt settings
 
 <!--/GITHUB_BLOCK-->
 
-The content of component contains from title and description blocks that can be set by the same properties names. To render custom content use `content` property.
+The `PlaceholderContainer` content is borrowed from the title and description sections, which you can set with the appropriate properties. To render custom content, use the `content` property.
```

Comment:
'is borrowed' breaks the meaning of sentence

Reply:
reverted to 'contains'